packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -217,8 +217,8 @@ export interface IContainerRuntimeOptions {
     // Delay before first attempt to spawn summarizing container
     initialSummarizerDelayMs?: number;
 
-    // Flag that enables running garbage collection to delete unused Fluid objects.
-    runGC?: boolean;
+    // Flag that will disable garbage collection if set to true.
+    disableGC?: true;
```

Comment:
Is the typing intentionally `true` only here?  Right now the only values allowed are `undefined` (or leaving unset since option) or `true`. I think it will prevent passing false.

Reply:
It is intentional. The idea is to set this flag only when you want to disable GC. Otherwise, don't pass it or leave it undefined. Do you think we should false as an option here?
Also, with the naming, it makes more sense to only have true as a value. disableGC: false is a double negation which seems unnecessary :-).

Reply:
Changed to boolean for clarity and readability.

Reply:
I think either way is fine, it's just some of the other code written around the typing seemed verbose.